settings/deno.vim

```diff
@@ -16,6 +16,18 @@ augroup vim_lsp_settings_deno
       \     'test': v:true,
       \     'testArgs': ['--allow-all'],
       \   },
+      \   "suggest": {
+      \     "autoImports": v:true,
+      \     "completeFunctionCalls": v:true,
+      \     "names": v:true,
+      \     "paths": v:true,
+      \     "imports": {
+      \       "autoDiscover": v:false,
```

Comment:
[memo]
https://github.com/denoland/vscode_deno/blob/main/docs/ImportCompletions.md#configuration
This setting is off at vim-lsp-settings.
Because if value is true, editor shoud promped like `add registry url?` and if select `yes`, editor should add registry url to `hosts` setting dynamic.
vim-lsp-setting does not provide such function.

Reply:
make sense